### ISSUE TITLE
[BUG] Custom fields with none value is raising an error

### ISSUE THREAD

**Issue by marcnil815** [LetMeR00t/TA-thehive-cortex#51]
### Request Type
Bug

### Work Environment

| Question              | Answer
|---------------------------|--------------------
| OS version (server)       |  9.x (Splunk Enterprise)
| TheHive version / git hash   | 3.0.1

### Problem Description
Custom fields of type "date" with a value of default 'None" are not caught.

### Steps to Reproduce
1. Create new custom field of type "date"
2. Create new Hive Case with the custom field, but leave empty
3. Listing events, cannot get past this Hive case due to arithmetic applied to a non-int value

### Possible Solutions
thehive_search_cases.py
```
120                         try:
121                             cf["value"] = time.strftime("%c %z",time.gmtime(int(cf["value"])/1000))
122                         except: pass
```

### Logs (issued from the search.log with logging mode set to DEBUG under Settings/Configuration)
-

**Comment by LetMeR00t** [comment]
Hello @marcnil815 ,
Thank you for this catch.
A fix will be provided in the v3.0.2

**Comment by LetMeR00t** [comment]
Hello
v3.0.2 was released on the Splunkbase but still waiting for the Cloud Vet.
If after installing this version, you keep having issues, please reopen this issue.
For now, I'm considering it done.
Thank you for your comprehension

**Comment by marcnil815** [comment]
Hi

I am still seeing the same issue here.
```
05-15-2023 10:56:58.207 ERROR ScriptRunner [1876253 phase_1] - stderr from '/pack/splunk/bin/python3.7 /pack/splunk/etc/apps/TA-thehive-cortex/bin/thehive_search_cases.py ad9bcbaa':  Traceback (most recent call last):
05-15-2023 10:56:58.207 ERROR ScriptRunner [1876253 phase_1] - stderr from '/pack/splunk/bin/python3.7 /pack/splunk/etc/apps/TA-thehive-cortex/bin/thehive_search_cases.py ad9bcbaa':    File "/pack/splunk/etc/apps/TA-thehive-cortex/bin/thehive_search_cases.py", line 121, in <module>
05-15-2023 10:56:58.207 ERROR ScriptRunner [1876253 phase_1] - stderr from '/pack/splunk/bin/python3.7 /pack/splunk/etc/apps/TA-thehive-cortex/bin/thehive_search_cases.py ad9bcbaa':      cf["value"] = time.strftime("%c %z",time.gmtime(int(cf["value"])/1000))
05-15-2023 10:56:58.207 ERROR ScriptRunner [1876253 phase_1] - stderr from '/pack/splunk/bin/python3.7 /pack/splunk/etc/apps/TA-thehive-cortex/bin/thehive_search_cases.py ad9bcbaa':  TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
05-15-2023 10:56:58.235 ERROR script [1876253 phase_1] - SearchMessage orig_component= sid=_eHJjYW5Ab3JzdGVkLmNvbQ_eHJjYW5Ab3JzdGVkLmNvbQ_VEEtdGhlaGl2ZS1jb3J0ZXg__search8_1684141016.830626 message_key=EXTERN:SCRIPT_NONZERO_RETURN__%s_%d_%s message=External search command 'thehivecases' returned error code 1. .
```

Testing on another system to verify...

**Comment by marcnil815** [comment]
Yes, confirmed on fresh system. For "NoneType" (empty date field), the code still  moves into the if sentence in line 121

**Comment by marcnil815** [comment]
@LetMeR00t cannot reopen this case :)

**Comment by LetMeR00t** [comment]
Hi @marcnil815 ,
Could you try to replace the mentionned lines with:

```
if "value" in cf and cf["value"] is not None:
     cf["value"] = time.strftime("%c %z",time.gmtime(int(cf["value"])/1000))
```

and let me know if it fixes your issue.

Thank you

**Comment by marcnil815** [comment]
@LetMeR00t 
That seemed to do the trick :)
I do noget get any errors on the empty custom fields anymore.

**Comment by LetMeR00t** [comment]
Hi @marcnil815 ,
Thank you, I'll add this fix in the future v3.0.3.
